components/button/button-copy.js

```diff
@@ -41,25 +42,38 @@ class ButtonCopy extends FocusMixin(LitElement) {
 
 	render() {
 		return html`
-			<d2l-button-icon icon="tier1:copy" text="Copy" @click="${this.#handleClick}"></d2l-button-icon>
-			<d2l-alert-toast ?open="${this._toastOpen}" @d2l-alert-toast-close="${this.#handleToastClose}">Copied!</d2l-alert-toast>
+			<d2l-button-icon ?disabled="${this.disabled}" icon="tier1:copy" text="${this.localize('intl-common:actions:copy')}" @click="${this.#handleClick}"></d2l-button-icon>
+			<d2l-alert-toast ?open="${this._toastMessage}" @d2l-alert-toast-close="${this.#handleToastClose}">${this._toastMessage}</d2l-alert-toast>
```

Comment:
Nit: I think I preferred `this._toastOpen` being a Boolean seeing as the message itself is always the same?

Reply:
There's still some discussion about displaying a different message when copy fails, which should be very rare. This change was made anticipating a different message, but I think the type of toast will also be different. So I was thinking something like `_toastState = undefined | copied | error` and then we localize the correct term and set the toast type in render based on that.